.changeset/spicy-masks-sort.md

```diff
@@ -0,0 +1,6 @@
+---
+"@firebase/firebase": patch
+"@firebase/analytics": patch 
+---
+
+Added Browser Extension check for Firebase Analytics. analytics.isSupported() will now return Promise<false> for extension environments. 
```

Comment:
Nit: backticks around `analytics.isSupported()` and `Promise<false>` will make it look nicer in markdown.